integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonLoadDataCommand.scala

```diff
@@ -837,7 +837,9 @@ case class CarbonLoadDataCommand(
           query = logicalPlan,
           overwrite = false,
           ifPartitionNotExists = false)
-      sparkSession.sparkContext.setLocalProperty(EXECUTION_ID_KEY, null)
+      if (!SparkUtil.isSparkVersionXandAbove("2.3")) {
+        sparkSession.sparkContext.setLocalProperty(EXECUTION_ID_KEY, null)
```

Comment:
Add a comment in all the 3 places about why this change is done.

`"spark.sql.execution.id is already set" exception will be thrown if not set to null in spark2.2 and below versions`

Reply:
@ajantha-bhat Handled the comment.